src/Uno.UI.Runtime.Skia.Wpf/WPF/WpfUIElementPointersSupport.cs

```diff
@@ -0,0 +1,299 @@
+﻿using System;
+using Windows.Devices.Input;
+using Windows.UI.Core;
+using Uno.Extensions;
+using Uno.Logging;
+using MouseEventArgs = System.Windows.Input.MouseEventArgs;
+using WpfApplication = System.Windows.Application;
+using WpfWindow = System.Windows.Window;
+using System.Threading;
+using System.Windows;
+using System.Windows.Input;
+using System.Windows.Interop;
+using System.Windows.Media;
+using Windows.UI.Input;
+using MouseDevice = System.Windows.Input.MouseDevice;
+using System.Reflection;
+
+namespace Uno.UI.Skia.Platform
+{
+	public class WpfUIElementPointersSupport : ICoreWindowExtension
+	{
+		private readonly ICoreWindowEvents _ownerEvents;
+		private readonly WpfHost _host;
+
+		private static int _currentFrameId;
+		private HwndSource _hwndSource;
```

Comment:
Codacy found an issue: [Remove the field '_hwndSource' and declare it as a local variable in the relevant methods.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5939197)